### ISSUE TITLE
convertWindowsTimeToDateTime returns wrong timezone for UTC time

### ISSUE THREAD

**Issue by vincemammoliti** [DirectoryTree/LdapRecord#515]
I am getting the whenChanged attribute of an AD user with $user->getFirstAttribute('whenChanged').

This returns a Carbon instance with the datetime matching what the time is in UTC, but incorrectly has the timezone set to my local timezone (Australia/Sydney).

e.g. Currently, if I var_dump the result of $user->getFirstAttribute('whenChanged') I will get:
```
["date"]=>
    string(26) "2022-10-21 04:30:00.000000"
["timezone"]=>
    string(16) "Australia/Sydney"
```
Whereas, I expect to get:
```
["date"]=>
    string(26) "2022-10-21 04:30:00.000000"
["timezone"]=>
    string(3) "UTC"


--- OR corrected for local timezone ---


["date"]=>
    string(26) "2022-10-21 15:30:00.000000"
["timezone"]=>
    string(16) "Australia/Sydney"

```

I believe one solution would be to provide a third argument specifying UTC timezone to the DateTime::createFromFormat() function.

In Timestamp.php:
```
protected function convertWindowsTimeToDateTime($value)
{
    return DateTime::createFromFormat(
        str_contains($value, '0Z') ? 'YmdHis.0\Z' : 'YmdHis.0T',
        $value,
        new \DateTimeZone('UTC')
    );
}
```

**Comment by stevebauman** [comment]
Hi @vincemammoliti,

Thanks for the report. Is your PHP timezone set to “Australia/Sydney” in your php.ini?

**Comment by vincemammoliti** [comment]
Yes it is

**Comment by stevebauman** [comment]
Thanks @vincemammoliti, I've just released [v2.19.0](https://github.com/DirectoryTree/LdapRecord/releases/tag/v2.19.0) with a fix for this bug. I appreciate the detailed report 🙏
